src/Bundle/Grid/Renderer/TwigBulkActionGridRenderer.php

```diff
@@ -14,34 +14,26 @@
 namespace Sylius\Bundle\ResourceBundle\Grid\Renderer;
 
 use Sylius\Bundle\ResourceBundle\Grid\Parser\OptionsParserInterface;
-use Sylius\Bundle\ResourceBundle\Grid\View\ResourceGridView;
 use Sylius\Component\Grid\Definition\Action;
 use Sylius\Component\Grid\Renderer\BulkActionGridRendererInterface;
 use Sylius\Component\Grid\View\GridViewInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 use Twig\Environment;
-use Webmozart\Assert\Assert;
 
 final class TwigBulkActionGridRenderer implements BulkActionGridRendererInterface
 {
-    private Environment $twig;
-
-    private OptionsParserInterface $optionsParser;
-
-    private array $bulkActionTemplates;
-
     public function __construct(
-        Environment $twig,
-        OptionsParserInterface $optionsParser,
-        array $bulkActionTemplates = [],
+        private Environment $twig,
+        private OptionsParserInterface $optionsParser,
+        private RequestStack $requestStack,
```

Comment:
Is that alright in terms of BC to add service without a BC layer? I didn't check if this is released code or something historical